libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Instrumentation/SFMethodInterceptor.m

```diff
@@ -121,7 +135,8 @@ - (id)forwardingTargetForSelector:(SEL)aSelector {
     } else {
         // Let's find out which interceptor instance is managing
         // this selector and class
-        SFMethodInterceptor *interceptor = InterceptorsForClassAndSelector[interceptorKey([self class], aSelector)];
+        BOOL isInstanceSelector = !(class_isMetaClass(object_getClass(self)));
```

Comment:
I just needed to stop and eat some lunch, turns out. ;-)  We can get the instance/class value by virtue of the calling class's context.